packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -240,22 +314,22 @@ class Tabs extends React.Component {
 
   render() {
     const {
-      tabData, ariaLabel, variant, onChange,
+      tabData, ariaLabel, variant, onChange, isDraggable,
```

Comment:
I think we need to replace all the reference of `tabdata` in render function to use State.visibileTabData to prevent possible difference in values between state and prop value